pkg/daemon/ceph/client/command.go

```diff
@@ -36,62 +34,41 @@ const (
 	cmdExecuteTimeout = 1 * time.Minute
 )
 
-func FinalizeCephCommandArgs(command string, args []string, configDir, clusterName string) (string, []string) {
-	// If the command should be run inside the toolbox pod, include the kubectl args to call the toolbox
-	if RunAllCephCommandsInToolbox {
```

Comment:
That makes sense. I'll have to change that. How does that actually work now that the toolbox is gone?

Reply:
The toolbox pod actually still exists as a convenient way to give you a pod where you can execute ceph commands. It simply launches the `rook/ceph` image instead of the old `rook/ceph-toolbox` image.